fastlane/metadata/android/release_notes.xml

```diff
@@ -1,170 +1,180 @@
+يمكنك الآن الاحتفاظ بالصوت عبر عنوان URL باستخدام مكوِّن الصوت.
+
+وأخيرًا، يمكن لقراء الشاشة الآن تحديد خلية السائر.
 </ar>
 <de-DE>
-17.4:
-Wir haben bei diesem Update 3 neue Updates für Geschichten hinzugefügt:
+17.5:
+Was ist neu in 17.5?
+
+Mit wiederverwendbaren Blöcken musst du Blockinhalte nur einmal erstellen. Dann kannst du sie direkt über den Block-Inserter beliebig oft wiederverwenden.
 
-Dir wird nun eine Medienauswahl angezeigt, wenn du einen Geschichtenblock vom Block-Editor hinzufügst, um dir den Start zu erleichtern.
+Schrägstrichbefehle erleichtern dir das Hinzufügen von Blöcken: Gib dazu einfach einen Schrägstrich gefolgt vom Blocknamen in den Editor ein.
 
-Die Geschichtenvorschau wird im Reader nun wie erwartet wiedergegeben.
+Mit dem Audioblock kannst du nun Audiodateien per URL einbinden.
 
-Und zuletzt versucht die App nun automatisch erneut, Slides hochzuladen, wenn dies fehlgeschlagen ist.
+Und nicht zuletzt können Screenreader ab sofort die Stepper-Zelle auswählen.
 </de-DE>
 <en-US>
-17.4:
-We’ve added 3 new updates to Stories in this update:
-
-You’ll now see a media picker when adding a Story block from the block editor, making it easier to get started.
+17.5:
+What’s new in 17.5?
 
-Story previews now render as expected in the Reader.
+Reusable blocks let you create block content once and reuse it infinite times, right from the block inserter.
 
-And finally, when slides fail to upload, the app will automatically retry, saving you an extra tap!
-</en-US>
-<es-ES>
-17.4:
-Hemos añadido 3 nuevas actualizaciones a las historias en esta actualización:
+Slash commands make adding blocks easy: just tap a forward slash and block name into the editor.
 
-Ahora verás un selector de medios cuando añadas un bloque de historias desde el editor de bloques, haciendo más fácil empezar.
+You can now pull in audio via URL with the Audio block.
 
-Las vistas previas de las historias ahora se muestran en el lector como se espera.
+Finally, screen readers can now select the stepper cell.
+</en-US>
+<iw-IL>
+17.5:
+מה חדש בגרסת 17.5?
 
-Y, por último, cuando las diapositivas fallan al subir, la aplicación volverá a intentarlo automáticamente, ¡ahorrándote un toque adicional!
-</es-ES>
-<fr-CA>
-17.4 :
-Nous avons ajouté 3 nouvelles mises à jour à Stories dans cette mise à jour :
+בלוקים לשימוש חוזר מאפשרים לך ליצור תוכן של בלוק פעם אחת ולהשתמש בו שוב ושוב, ללא הגבלה, ישירות מתוך מוסיף הבלוקים.
 
-Vous verrez désormais un sélecteur de média lors de l’ajout d’un bloc Story dans l’éditeur de blocs, ce qui facilite le démarrage.
+פקודות של קו נטוי מאפשרות לך להוסיף בלוקים בקלות: אפשר להקיש על הקו הנטוי לפנים ולהקליד את שם הבלוק בעורך.
 
-Les aperçus de story ont désormais le rendu attendu dans le Lecteur.
+אפשר להוסיף אודיו דרך כתובת URL באמצעות בלוק האודיו.
 
-Enfin, lorsque le chargement des diapositives échoue, l’application réessaie automatiquement, ce qui vous économise un appui supplémentaire !
-</fr-CA>
-<fr-FR>
```

Comment:
Those release notes are missing some of the Mag 16, namely `fr-FR` and `es-ES`.

Since then, [Both `fr-FR` and `es-ES` release notes are now available in GlotPress](https://translate.wordpress.org/projects/apps/android/release-notes/fr/default/); so could it be worth including them in the PlayStore's XML before hitting publish? I think we can edit release details / metadata without having to go through review again, as long as we don't change the binary… right?

Reply:
Oh nvm, I tried updating and saw the issue:
```
$ fl download_metadata_strings version:17.5 build_number:1058
…
[12:54:26]: es-ES translation for release_note_175 exceeds maximum length (501). Switching to the alternate translation.
…
[12:54:27]: fr-CA translation for release_note_175 exceeds maximum length (520). Switching to the alternate translation.
…
[12:54:28]: fr-FR translation for release_note_175 exceeds maximum length (520). Switching to the alternate translation.
```
I guess I'll play it safe and merge as is for this one then (esp since it's my first release of WPAndroid, don't want to mess things up)

Reply:
>  I think we can edit release details / metadata without having to go through review again, as long as we don't change the binary… right?

Correct!